utils/statediff/src/lib.rs

```diff
@@ -124,46 +120,41 @@ fn pp_actor_state(
     depth: Option<u64>,
 ) -> Result<String, anyhow::Error> {
     let resolved = actor_to_resolved(bs, state, depth);
-    let ipld = &resolved.state.0;
     let mut buffer = String::new();
-
     writeln!(&mut buffer, "{state:?}")?;
-
-    // FIXME: Use the actor interface to load and pretty print the actor states.
-    //        Tracker: https://github.com/ChainSafe/forest/issues/1561
-    if let Ok(miner_state) = forest_ipld::from_ipld::<MinerState>(ipld.clone()) {
+    if let Ok(miner_state) = MinerState::load(bs, &state.into()) {
         write!(&mut buffer, "{miner_state:?}")?;
         return Ok(buffer);
     }
-    if let Ok(cron_state) = forest_ipld::from_ipld::<CronState>(ipld.clone()) {
+    if let Ok(cron_state) = CronState::load(bs, &state.into()) {
```

Comment:
@sudo-shashank Have you confirmed that this works locally? Sadly, we have no unit tests for now.

Reply:
Not yet

Reply:
Add the missing actors and I will write unit tests.